### ISSUE TITLE
Questions about slides

### ISSUE THREAD

**Issue by gituser0424** [trusthlt/nlp-with-deep-learning-lectures#8]
Slide 17: Regarding factorizing the joint probability into a product in the bottom of the slide, 
should w_k in Pr(w_k|<S>,w_1,w_2,....,w_n_1) be w_n? 

Slide 25: You mentioned that embedding vector for each word is part of data to be trained. On the other hand, I am not able to see how embedding is part of trained data. This is because I am not able to see the green "E box" is involved in the training process. Note that "concat" is in purple (which means to be non-trained data). I understand that the network can learn the weights associated with the embedding vector. Then, how are such weights related to the embedding vector?

**Comment by habernal** [comment]
As of your first question, can you please be more specific which lecture? (you can also point to the actual line in the actual file here)

**Comment by gituser0424** [comment]
Both questions refer to NLP with Deep Learning 05 - Text generation 1: Language models and word embeddings. Uploaded two screenshots for the above two questions. Thank you.
![embedding](https://github.com/trusthlt/nlp-with-deep-learning-lectures/assets/33124139/5847d0ee-027e-43c9-8735-c34656d0c0b5)
![wk](https://github.com/trusthlt/nlp-with-deep-learning-lectures/assets/33124139/554bd510-f3c3-4e60-aee0-2d11a0ebdf40)

**Comment by habernal** [comment]
Thanks for the catch, fixed in #9

**Comment by habernal** [comment]
> Slide 25: You mentioned that embedding vector for each word is part of data to be trained. On the other hand, I am not able to see how embedding is part of trained data. This is because I am not able to see the green "E box" is involved in the training process. Note that "concat" is in purple (which means to be non-trained data). I understand that the network can learn the weights associated with the embedding vector. Then, how are such weights related to the embedding vector?

Training data are tuples (input features, label), which are (indices of previous words, index of the target word). All green boxes are trainable parameters (or "weights") that are updated during SGD (and backprop is used for computing the gradient efficiently). All lightblue boxes are differentiable functions that have input arguments, which might be parameters or output of previous nodes. The whole structure is a differentiable function represented as a computational graph (DAG). If this is not clear enough, go back to the DAG example from the previous lecture and consult Yoav Goldberg's book (referenced thorough the slides).
